lc-hax/Scripts/Commands/Privileged/SpawnCommand.cs

```diff
@@ -0,0 +1,63 @@
+using GameNetcodeStuff;
+using Hax;
+using System;
+using System.Collections.Generic;
+using Unity.Netcode;
+using UnityEngine;
+using UnityObject = UnityEngine.Object;
+
+[PrivilegedCommand("/spawn")]
+public class SpawnCommand : ICommand {
+    private void SpawnEnemyOnPlayer(PlayerControllerB player, GameObject prefab, ulong amount = 1) {
+        if (Helper.RoundManager == null) return;
+        if (player == null) return;
+
+        for (ulong i = 0; i < amount; i++) {
+            GameObject enemy = UnityEngine.Object.Instantiate(prefab, player.transform.position, Quaternion.Euler(Vector3.zero));
```

Comment:
I don't understand how you see that you have left two different ways to write the same thing and think "let's merge this!"

```suggestion
            GameObject enemy = UnityObject.Instantiate(prefab, player.transform.position, Quaternion.Euler(Vector3.zero));
```